platform/linux-generic/pktio/dpdk.c

```diff
@@ -93,6 +95,56 @@ void refer_constructors(void)
 }
 #endif
 
+static void lookup_opt(config_setting_t *default_opt, config_setting_t *drv_opt,
+		       const char *opt, int *val)
+{
+	/* Default option */
+	config_setting_lookup_int(default_opt, opt, val);
+
+	/* Driver option overwrites default option */
+	if (drv_opt)
+		config_setting_lookup_int(drv_opt, opt, val);
+}
+
+static void init_options(pktio_entry_t *pktio_entry,
+			 const struct rte_eth_dev_info *dev_info)
+{
+	dpdk_opt_t *opt = &pktio_entry->s.pkt_dpdk.opt;
+	config_setting_t *default_opt;
+	config_setting_t *drv_opt;
+
+	/* Default values. Update 'config/odp-linux.conf' if modified. */
+	opt->num_rx_desc = DPDK_NM_RX_DESC;
+	opt->num_tx_desc = DPDK_NM_TX_DESC;
+	opt->rx_drop_en = 0;
+
+	default_opt = _odp_libconfig_lookup("pktio_dpdk");
+	if (!default_opt) {
+		ODP_DBG("No DPDK pktio options found\n");
+		goto done;
+	}
+
+	/* config_lookup_from() was renamed to config_setting_lookup() in
+	 * libconfig 1.5.0 */
+#if (LIBCONFIG_VER_MAJOR <= 1 && LIBCONFIG_VER_MINOR < 5)
```

Comment:
According to the [libconfig changelog](https://github.com/hyperrealm/libconfig/blob/master/ChangeLog) there are versions pre-1.0 (e.g., 0.9) which would fail this test. This needs to be reversed so that you use the newer form for v1.5 and higher levels:
```
#if (LIBCONFIG_VER_MAJOR > 1 || (LIBCONFIG_VER_MAJOR == 1 && LIBCONFIG_VER_MINOR >= 5)) ...
```

Reply:
Will fix.